core/templates/dev/head/domain/AnswerStatsFactory.js

```diff
@@ -0,0 +1,65 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Factory for domain object which holds the statistics of a
+ * particular answer from some particular state.
+ */
+
+oppia.factory('AnswerStatsFactory', [function() {
```

Comment:
Hm. Just to check, how do you feel about AnswerStat (singular)? (There's only one answer here and one stat -- its frequency.)

Reply:
Hmm, I've been considering "isAddressed" as one of the statistics of an answer.

Reply:
Ah ok. Yeah I think it's fine. Thanks!